src/components/checkout/CheckOutForm.js

```diff
@@ -0,0 +1,179 @@
+const InputEmail = styled.input`
+  width: 21vw;
+  @media (max-width: 900px) {
+    width: 65vw;
+  }
+`;
+
+const LabelZipCode = styled.label`
+  margin-bottom: 0.3vw;
+`;
+const InputZipCode = styled.input`
+  width: 21vw;
+  @media (max-width: 900px) {
+    width: 65vw;
+  }
+`;
+
+const LabelInstructions = styled.label`
+  margin-bottom: 0.3vw;
+`;
+const InputInstructions = styled.textarea`
+  resize: none;
+  width: 21vw;
+  min-height: 10vw;
+  @media (max-width: 900px) {
+    width: 65vw;
+  }
+`;
+
+const Separator = styled.div`
+  margin-bottom: 1vw;
+  display: flex;
+  flex-direction: column;
+`;
+const Total = styled.p`
+  text-align: right;
+`;
+
+export default function CheckOutForm() {
+  window.scrollTo(0, 0);
+  const { cartItems } = useContext(ListContext);
+
+  const [form, setForm] = useState([]);
+  const handleChange = (e) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return (
+    <>
+      <Space></Space>
+      <Container>
+        <Title>Checkout</Title>
+        <Wrapper>
+          <FormTitle>Add your Information</FormTitle>
+          <CheckoutForm>
+            <FormWrapper>
+              <Separator>
+                <LabelName htmlFor="name">Name: </LabelName>
+                <InputName
+                  name="name"
+                  id="name"
+                  onChange={handleChange}
+                ></InputName>
+              </Separator>
+              <Separator>
+                <LabelEmail htmlFor="email">Email: </LabelEmail>
+                <InputEmail
+                  name="email"
+                  id="email"
+                  onChange={handleChange}
+                ></InputEmail>
+              </Separator>
+              <Separator>
+                <LabelZipCode htmlFor="zip">Zip Code: </LabelZipCode>
+                <InputZipCode
+                  name="zip"
+                  id="zip"
+                  onChange={handleChange}
+                ></InputZipCode>
+              </Separator>
+              <Separator>
+                <LabelInstructions htmlFor="instructions">
+                  Delivery Instructions:
+                </LabelInstructions>
+                <InputInstructions
+                  name="instructions"
+                  id="instructions"
+                  onChange={handleChange}
+                ></InputInstructions>
+              </Separator>
+            </FormWrapper>
+          </CheckoutForm>
+        </Wrapper>
+      </Container>
+      <ProductsContainer>
+        <ShoppingProd checkout={true}></ShoppingProd>
+        <Total>
+          <b>
+            Total: $
+            {cartItems.length
+              ? cartItems
+                  .map((item) => item.data.price * item.qty)
+                  .reduce((acc, curr) => acc + curr)
+              : 0}
+          </b>
+        </Total>
+      </ProductsContainer>
+      <CheckOutButtons></CheckOutButtons>
+      <Space></Space>
+      <Space></Space>
```

Comment:
when the elements don't have children, you can omit the closing tag, this removes the noise and it is easier to read the component

```jsx
<CheckOutButtons />
<Space />
<Space />
```